### PR TITLE
Register Page

### PR DESCRIPTION
**Pull request by zeetec20**
Waiting for merge #37 to continue progress

### CODE REVIEW

**Comment by zeetec20** [comment]
Change component select with Headless UI

File: app/(auth)/register/page.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+import { DataSelect } from '~/components/ui/form/select/types'
+import TextInput from '~/components/ui/form/text-input'
+import TextAreaInput from '~/components/ui/form/textarea'
+import countries from '~/data/countries.json'
+import clsxm from '~/utils/clsxm'
+
+const RegisterPage = () => {
+  const [isSubmit, setIsSubmit] = useState(false)
+  const [country, setCountry] = useState<DataSelect | undefined>()
+  // const [country, setCountry] = useState<string>()
```
Comment (kalwabed) [inline app/(auth)/register/page.tsx:19]:
```suggestion
```

File: app/(auth)/register/page.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  return (
+    <div className="flex min-h-screen w-full flex-col items-center justify-center bg-custom-white-broken">
+      <div className="mt-8 mb-9 flex items-center justify-center overflow-hidden rounded-full bg-white p-3 shadow-md">
+        <Image src={Logo} alt="logo site" width={75} height={75} />
+      </div>
+
+      <div className="container flex flex-row justify-center">
+        <form
+          className={clsxm(
+            'bg-w group h-min basis-1/2 rounded border-t-2 border-t-custom-purple bg-white shadow-sm',
```
Comment (kalwabed) [inline app/(auth)/register/page.tsx:35]: `bg-w`? `h-min`?
Did you try using some plugins or what?

**Review by kalwabed — CHANGES_REQUESTED** [review]

File: app/(auth)/register/page.tsx (hunk 2 above)
Comment (zeetec20) [reply app/(auth)/register/page.tsx:35]: Okay I will check it

**Review by kalwabed — APPROVED** [review]
LGTM :100:

**Comment by zeetec20** [comment]
Fix #39 
the component has been created at register page

**Pull request merged**